server/controllers/crm/crm7Controller.ts

```diff
@@ -16,6 +16,14 @@ export default class Crm7Controller {
       const usn = Number(req.params.usn)
       const { sectionId } = req.params
       const crm7Response = await this.crm7Service.getCrm(usn, getProfileAcceptedTypes(res))
+
+      const mergedScheduleCostsData = {
+        ...crm7Response.formDetails.scheduleOfTimeSpent,
+        ...crm7Response.formDetails.claimOfCosts,
```

Comment:
I don't think we need to merge scheduleOfTimeSpent & claimOfCosts. As all the data should be coming from scheduleOfTimeSpent.